### DISCUSSION TITLE
How to report a bug with my computerwith linux mint cinamon

### DISCUSSION THREAD

**Discussion by fredlinux83** [community/community#28747] [Other Feature Feedback, Questions, & Ideas]
Hello,
recent user of linux mint.
Having a problem with cinamon and was told on forum to report it there (i'm not fluent in english and not sure this is the correct place).
Problem description: cpu over use by some cinamon -replace seen on htop.
Htop report here:
https://dpaste.com/C3GF6Y6A9
Hope someone will see where the problem is.
Tanks.

**Comment by airtower-luna** [comment]
> not sure this is the correct place

I'm afraid not. This is the GitHub Community, mostly for general discussions around Git and GitHub. You should report your problem to the Cinnamon developers.

I looked at the [Linux Mint Troubleshooting Guide](https://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/). It looks like the right place to report Cinnamon bugs is the https://github.com/linuxmint/Cinnamon repository. I recommend you look at the troubleshooting guide yourself to see what they'd like to see in a bug report.

Good luck! :slightly_smiling_face:
